.github/workflows/ci.yml

```diff
@@ -154,6 +160,7 @@ jobs:
       uses: actions/setup-python@v5
       with:
         python-version: ${{ matrix.python }}
+        allow-prereleases: true
```

Comment:
Why not using `"3.13-dev"` instead to better indicate that Python 3.13 is not yet available as a stable release?

Reply:
Feels easier to let it stay as `3.13` after the final release rather than coming back and changing it from `3.13-dev` to `3.13`. I don't have a strong preference, though.